.claude/CODE-COMPLEXITY-MONITOR.md

```
@@ -0,0 +1,286 @@
+
+### Automatic Triggers
+
+The skill activates automatically after:
+- ✅ Edit operations on code files
+- ✅ Write operations creating new files
+- ✅ Manual requests: "check complexity", "analyze this"
+
+### Example Usage
+
+**Automatic (after editing code)**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)